File: Tennis.Play/ISide.cs
using System;

namespace Tennis.Play
{
	public interface ISide
	{
		string TeamName { get; set; }
		
		int Strength { get; set; }
		
		bool IsServing { get; set; }
		
		int ServingStrength { get; set; }
		
		int ReturningStrength { get; set; }
	}
}

File: Tennis.FSharp.Play/ISide.fs
namespace Tennis.FSharp.Play

    type ISide =
        abstract member TeamName : string
            with get, set
        abstract member Strength : int
            with get, set
        abstract member IsServing : bool
            with get, set
        abstract member ServingStrength : int
            with get, set
        abstract member ReturningStrength : int
            with get, set
